recipes/fuse-overlayfs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: fuse-overlayfs
+  version: {{ version }}
+
+source:
+  - url: https://github.com/containers/fuse-overlayfs/archive/v{{ version }}.tar.gz
+    sha256: 5df218732244059057686194b0e1fef66fb822d4087db48af88e1bc29bb1afde
+    patches:
+      - 0001-Add-missing-defines-for-CentOS-6.patch
+      - 0002-Fix-version-tag.patch
```

Comment:
Maybe add a comment saying this needs to be updated when the version is? Better yet, can it be modified to use the `$PKG_VERSION` environment variable?

Reply:
No need to. I was even stricter and make the build fail, meaning, I check for it in `build.sh` ;).
(And I prefer the patch here as a reminder for myself.)